source/_components/blink.markdown

```diff
@@ -25,6 +25,8 @@ blink:
   password: YOUR_PASSWORD
 ```
 
+If you have your camera registered .homeassistant/deps/blinkpy.py file and change .prod to .prde to access the EU or other region server for Blink to work.
```

Comment:
This looks more like something that should be handled in the code. If a user make this change then it will be gone with the next update.